L1Trigger/CSCTriggerPrimitives/plugins/CSCTriggerPrimitivesAnalyzer.cc

```diff
@@ -0,0 +1,204 @@
+#include "L1Trigger/CSCTriggerPrimitives/plugins/CSCTriggerPrimitivesAnalyzer.h"
```

Comment:
Unit tests to test the functionalities of your Library and/or Plugins should go under .../test.
See https://cms-sw.github.io/cms_coding_rules.html

Reply:
All analyzers moved to test/. Analyzers being deprecated put in test/deprecated/ for now until the remaining useful pieces are integrated